.azure/hpu-tests.yml

```diff
@@ -104,40 +104,38 @@ jobs:
         python -m pytest -sv \
           test_fabric/test_hpu_accelerator.py \
           test_fabric/test_hpu_strategy.py \
+          --hpus 1 --junitxml=hpu_test-fabric-results.xml
+        python -m pytest -sv \
           test_pytorch/test_accelerator.py \
-          test_pytorch/test_deepspeed.py \
           test_pytorch/test_hpu_graphs.py \
           test_pytorch/test_dynamic_shapes.py \
-          --forked --junitxml=hpu_test-results.xml
-      workingDirectory: tests/
-      displayName: 'General HPU test'
-
-    - bash: |
-        python -m pytest -sv \
           test_pytorch/test_datamodule.py \
           test_pytorch/test_profiler.py \
-          --forked --hpus 1 --junitxml=hpu1_test-results.xml
+          --hpus 1 --junitxml=hpu_test-torch-results.xml
       workingDirectory: tests/
-      displayName: 'Single card test'
+      displayName: 'HPU General tests'
 
     - bash: |
         set -ex
-        python -m pytest -sv  test_pytorch/test_accelerator.py \
-          --forked --hpus 2 --junitxml=hpu2_accel_test-results.xml
-        python -m pytest -sv test_pytorch/test_profiler.py \
-          --forked --hpus 2 --junitxml=hpu2_profiler_test-results.xml
+        python -m pytest -sv test_pytorch/test_deepspeed.py \
+        --junitxml=hpu_deepspeed_test-results.xml
       workingDirectory: tests/
-      displayName: 'Multi card(2) HPU test'
+      displayName: 'HPU Deepspeed tests'
 
     - bash: |
         LOWER_LIST=test_pytorch/ops_fp32.txt \
           FP32_LIST=test_pytorch/ops_bf16.txt \
           python -m pytest -sv test_pytorch/test_precision.py \
           -k test_autocast_operators_override --runxfail \
-          --forked --junitxml=hpu_precision_test_override-results.xml
+          --junitxml=hpu_precision_test_override-results.xml
       workingDirectory: tests/
       displayName: 'HPU precision test'
 
+    - bash: |
+        set -ex
```

Comment:
The same here, it make sense only with multiple commands

Reply:
taken care in #119